api-errors.json

```diff
@@ -12,51 +12,51 @@
         "long_message": "The request was invalid. The client should not repeat this request without modification"
     },
     "40002": {
-        "short_message": "Redirect URI mismatch",
+        "short_message": "Redirect URI Mismatch",
         "long_message": "The redirect_uri does not match the registered redirect_uri"
     },
     "40003": {
         "short_message": "Invalid Scope",
         "long_message": "An invalid or insufficient scope was used"
     },
     "40004": {
-        "short_message": "Already Registered",
+        "short_message": "User Already Registered",
         "long_message": "This user is already registered"
     },
     "40005": {
-        "short_message": "Error in phonenumber",
+        "short_message": "Error In Phonenumber",
```

Comment:
I wonder if `PhonenumberError` would be better than `ErrorInPhonenumber`.

Same for `4006`, `4009`.

The change would be similar to the one you already introduced for `40012`